File: Calculate mean and concatenate string.py
def mean(lst):
    filterd_n = [int(i) for i in lst if i.isdigit()]
    filterd_s = [i for i in lst if not(i.isdigit())]
    return [sum(filterd_n)/len(filterd_n),"".join(filterd_s)]
    

File: Calculate mean and concatenate string.ts
export function mean(lst: string[]): [number, string] {
  const filterd_n = lst.filter((tmp:string) => !isNaN(Number(tmp))).map(tmp => parseInt(tmp))
  const filterd_s = lst.filter((tmp:string) => isNaN(Number(tmp)))
  return [filterd_n.reduce((a,b) => Number(a)+Number(b),0)/10,filterd_s.join("")]
}
